app/package/controllers.py

```diff
@@ -133,10 +131,6 @@ def delete_data_package(publisher, package):
         status_db = Package.change_status(publisher, package, PackageStateEnum.deleted)
         if status_acl and status_db:
             return jsonify({"status": "OK"}), 200
-        if not status_acl:
```

Comment:
Again I think we do want to know if this failed. We can't remove the exception handling here.